index.d.ts

```diff
@@ -0,0 +1,52 @@
+// TypeScript typings for "mocha-testdata@1.1.3".
+
+declare module "mocha-testdata" {
```

Comment:
No, this is the scoping container that prevents the types inside to "leak" into the ambient scope.  At least that is my understanding.